types/globalize/dist/globalize.d.ts

```diff
@@ -41,7 +41,11 @@ declare namespace Globalize {
 		/**
 		 * Boolean (default is true) value indicating whether a grouping separator should be used.
 		 */
-		useGrouping?: boolean;
+        useGrouping?: boolean;
```

Comment:
The indentation here seems to be off from the rest of the file.

Reply:
Fixed